spec/unit/provider/exec/pwsh_spec.rb

```diff
@@ -114,6 +131,45 @@
           expect(status.exitstatus).to eq(1)
         end
       end
+
+      context "on non-Windows", :if => !Puppet.features.microsoft_windows? do
+        # The usage of uname is a little fragile however there is basically nothing
+        # which is universal across all Linux/Unix/Mac distributions; Unlike Well Known SIDS in Windows
+        # The closest is the presence of the uname command and its generic text output
+        let(:command) { '& uname' }
```

Comment:
Is it important to the test hat we shell out to a command like this?

Reply:
In this case we're testing stdin/stdout streams are being passed from external programs back to us in the provider.  This is really only important on the PowerShell Manager side of things, but still "useful" for the script based invocations.